DivisionManagementSystem/settings.py

```diff
@@ -84,6 +84,29 @@
 
 WSGI_APPLICATION = 'DivisionManagementSystem.wsgi.application'
 
+# Logging
+
```

Comment:
Block comments should not be followed by a newline, they should directly precede the block.